### PR TITLE
Cloudbroker scenarios testcases

### PR DESCRIPTION
**Pull request by Dinaamagdy**

### CODE REVIEW

**Comment by Dinaamagdy** [comment]
```
dina@dina-Inspiron-5423:~/restapi/latest/G8_testing/functional_testing/Openvcloud/RESTful$ nosetests -v -s testcases/system/test01_usermanager.py --tc-file config.ini
OVC-001 [with case='exists', response_code=200] ... ok
OVC-001 [with case='non-exist', response_code=404] ... ok
OVC-002 [with *args=('exists', 200)] ... SKIP: https://github.com/0-complexity/openvcloud/issues/1367
OVC-002 [with *args=('non-exist', 404)] ... SKIP: https://github.com/0-complexity/openvcloud/issues/1367
OVC-001 [with case='exists', response_code=200, userexists='true'] ... ok
OVC-001 [with case='non-exist', response_code=404, userexists='false'] ... ok
OVC-002 [with case='exists', response_code=200] ... ok
OVC-002 [with case='non-exist', response_code=404] ... ok
OVC-003 [with case='exists', response_code=200] ... ok
OVC-003 [with case='non-exist', response_code=404] ... ok

----------------------------------------------------------------------
Ran 10 tests in 36.662s

```

File: functional_testing/Openvcloud/RESTful/framework/api/cloudbroker/cloudspace.py [hunk 1]
```diff
@@ -80,7 +80,7 @@ def destroy(self, cloudspaceId, accountId, **kwargs):
         reason = kwargs.get('reason',utils.random_string())       
         return self._api.cloudbroker.cloudspace.deployVFW(accountId=accountId, cloudspaceId=cloudspaceId, reason=reason)
 
-    def destroyCloudSpaces(self, cloudspaceIds, **kwargs):
+    def destroycloudspaces(self, cloudspaceIds, **kwargs):
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/framework/api/cloudbroker/cloudspace.py]: rename function name to be like the api name

File: functional_testing/Openvcloud/RESTful/framework/utils/ovc_client.py [hunk 2]
```diff
@@ -61,4 +65,7 @@ def load_swagger(self):
                 :type %(name)s: %(type)s""" % param
             api.__doc__ = docstring
 
-        return swagger
\ No newline at end of file
+        return swagger
+
+    def lg(self, msg):
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/framework/utils/ovc_client.py]: why you add log function in ovc_client file ?

File: functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py [hunk 3]
```diff
@@ -0,0 +1,306 @@
+from testcases import *
+from nose_parameterized import parameterized
+import random, unittest
+#from framework.api.client import Client
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py]: remove this comment

File: functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+
+
+
+class Test(TestcasesBase):
+
+    def setUp(self):
+        super().setUp()
+        self.user="{}@itsyouonline".format(self.whoami)
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py]: username should be passed in config file in this format ```username@itsyouonine```
so remove @itsyouonline and just use self.whoami

File: functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py [hunk 5]
```diff
@@ -0,0 +1,117 @@
+from testcases import *
+from nose_parameterized import parameterized
+
+
+class cloudspace(TestcasesBase):
+
+    @classmethod
+    def setUpClass(cls):
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py]: remove this function if you don't use it

File: functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+        super().setUpClass()
+
+    @classmethod
+    def tearDownClass(cls):
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py]: remove this function if you don't use it

File: functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+        self.lg(" [*] Create account")
+        self.user = "{}@itsyouonline".format(self.whoami)
+        self.account, self.response = self.api.cloudbroker.accounts.create(self.user)
+        self.assertEqual(self.response.status_code, 200)
+
+        self.lg(" [*] Create cloudspace.")
+        self.api.cloudbroker.cloudspaces.create(accountId=self.account, location=self.api.get_random_locations(),
+                                                access=self.user)
+
+    def tearDown(self):
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py]: remove this function if you don't use it

File: functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+    def test001_addExtraIP(self):
+        """ OVC-000
+        *Test case for adding user to account with read access.*
+
+        **Test Scenario:**
+        #. Create Cloudspace [cs1].
+        #. Add wrong IP, should fail.
+        #. Add Extra IP on cloudspace[cs1], should succeed.
+        #. Check thar IP added to cloudspace [CS1], should succeed.
+        """
```
Comment (ahmedelsayed-93) [inline functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test02_cloudspaces.py]: add pass to skip this method

**Pull request merged**